data/hyperOutportBlocks/hyperOutportBlock.proto

```diff
@@ -18,11 +20,23 @@ message BlockHashRequest {
   string hash = 1;
 }
 
+message BlockHashStreamRequest {
+  string hash = 1;
+  google.protobuf.Duration pollingInterval = 2;
```

Comment:
i think it might be better to set the polling interval based on the config, not based on the client request, since the client might not have good insight on how to set the interval, as it should be below our round time, otherwise it will constantly stay behind

Reply:
it can be done in a next PR

Reply:
since the polling is somewhat independent of the server. I thought it would be a good idea to have it as such.